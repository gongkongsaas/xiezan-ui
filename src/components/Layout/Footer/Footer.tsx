import { defineComponent } from 'vue'

export default defineComponent({
  setup() {
    return () => <footer></footer>
  },
})
