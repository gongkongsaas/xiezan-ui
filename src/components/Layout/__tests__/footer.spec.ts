import { shallowMount } from '@vue/test-utils'
import Footer from '../footer'

describe('footer', () => {
  it('footer is mounted', () => {
    const wrapper = shallowMount(Footer)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
