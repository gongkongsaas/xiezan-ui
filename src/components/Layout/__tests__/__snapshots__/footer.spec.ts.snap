// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer footer is mounted 1`] = `<footer class="footer"></footer>`;
